From ba124c120b0ad49da11bba78a5830a0bf825a858 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@linaro.org>
Date: Tue, 10 Jan 2023 22:38:28 +0100
Subject: [PATCH 0026/1695] plat-stm32mp1: enable async notif on stm32mp13

Enables async notif using GIC PPI 15 as non-secure interrupt notifier
for STM32MP13 variants.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d3309119a05a1bd0868c9a978e23c14cbd4a00bc plat-stm32mp1: enable async notif on stm32mp13]
Change-Id: Iba7951516e009c6a5ca2f3d4ed20a66f64f81b16
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
[etienne: cherry-picked from 91d8d7b716c9d17c0f1a88679b9eb121be5a3885]
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 3f981361e..48db504de 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -106,6 +106,8 @@ $(call force,CFG_REGULATOR_FIXED,y)
 
 ifeq ($(CFG_STM32MP13),y)
 $(call force,CFG_BOOT_SECONDARY_REQUEST,n)
+$(call force,CFG_CORE_ASYNC_NOTIF,y)
+$(call force,CFG_CORE_ASYNC_NOTIF_GIC_INTID,31)
 $(call force,CFG_CORE_RESERVED_SHM,n)
 $(call force,CFG_DRIVERS_CLK_FIXED,y)
 $(call force,CFG_SECONDARY_INIT_CNTFRQ,n)
-- 
2.47.2

